### ISSUE TITLE
Adding some basic doc editing features

### ISSUE THREAD

**Issue by praneeth-rdy** [Praneeth-rdy/cvGenerator#2]
## TODO
- Bold Text
- Links addition
- Image Upload or Replace the existing one

Read about rich-text fields before working on the above feature improvements.

**Comment by j-tesla** [comment]
Bold text works with `Ctl+B`

**Comment by praneeth-rdy** [comment]
Yeah!! I haven't noticed. Ig it is provided by default when "contenteditable" is true.
